server/src/controllers/User/resolver.ts

```diff
@@ -49,6 +55,58 @@ export class UserWithInstanceRoleResolver {
     });
   }
 
+  @Query(() => UserInformation)
+  async userInformation(
+    @Ctx() ctx: ResolverCtx,
+  ): Promise<UserInformation | null> {
+    if (!ctx.user) {
+      return null;
+    }
+    return await prisma.users.findFirst({
```

Comment:
```suggestion
    return await prisma.users.findUnique({
```
Nitpick: this is more declarative since there can only be a single user record with this id.